frontend/src/features/public-form/components/FormPaymentPage/stripe/components/DownloadReceiptBlock.tsx

```diff
@@ -198,7 +200,8 @@ export const DownloadReceiptBlock = ({
             leftIcon={<BiDownload fontSize="1.5rem" />}
             onClick={onClick || handleInvoiceClick}
           >
-            Save proof of payment
+            {t('features.common.save')}{' '}
+            {t('features.common.entities.paymentProof')}
```

Comment:
Similar comments as above; to avoid concatenated strings when dealing with translations.